src/components/lyne-teaser/lyne-teaser.tsx

```diff
@@ -0,0 +1,109 @@
+import {
+  Component,
+  h,
+  Prop,
+  Watch
+} from '@stencil/core';
+
+import { InterfaceTeaserAttributes } from './lyne-teaser.custom';
+
+@Component({
+  shadow: true,
+  styleUrls: {
+    default: 'styles/lyne-teaser.default.scss',
+    shared: 'styles/lyne-teaser.shared.scss'
+  },
+  tag: 'lyne-teaser'
+})
+
+/**
+ * Generalized product Teaser — merge of ticket and subscription
```

Comment:
For me the comment here is still not clear, what means "merge of ticket and subscription". Would be possible to describe more?